### DISCUSSION TITLE
Guide to use v3 or v2 for new projects

### DISCUSSION THREAD

**Discussion by surfmuggle** [xunit/xunit#3103] [Question] [answered]
The [v3 release note](https://xunit.net/releases/v3/1.0.0) contains an information that v2 users can migrate to v3. 

 - Should a new project start with v3 or should it stay with v2 for a while due to possible bumps using v3.0? 
 - If there is a recommendation for new users to use v3 or v2 it would be great if this could be added to the [README.md](https://github.com/xunit/xunit/blob/main/README.md).

**Comment by bradwilson** [comment, marked as answer]
> Should a new project start with v3 or should it stay with v2 for a while due to possible bumps using v3.0?

That's up to you to decide. Obviously our focus is 100% on v3 moving forward, so you should expect there will be 
no new features or bug fixes for v2, so starting with v3 is likely to be your best future-proof decision.

There are other factors to your decision right now that may suggest you choose v2 vs. v3, including any third party libraries or extensions to xUnit.net that you're using, and whether they're compatible with v3 yet. Since v3 is relatively new, it's likely that some third party libraries will be keeping you on v2 for a little while yet.

> If there is a recommendation for new users to use v3 or v2 it would be great if this could be added to the [README.md](https://github.com/xunit/xunit/blob/main/README.md).

Once we have made the firm decision to fully deprecate v2, we will update the README to make that known.
